magento/magento2
# Travis CI functional tests maintenance for 2.2-develop

### Description
Current state of Travis CI builds for branch `2.2-develop` is not equal with `2.3-develop`. Functional test suite is taking too long to execute and jobs usually finish with the timeout.

This PR ports a fix, which was applied to `2.3-develop`, reducing the amount of tests executed on the Travis CI. Only high severity scenarios will be executed.

Whole functional tests suite is always executed per each pull request with the Magento's in-house CICD system during the merge process, ensuring all of the tests are green, thus reducing the amount of tests executed on Travis would not influence quality.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios
1. Build Travis CI with this PR applied
2. Check sanity of the tests execution (logs, artifacts for each job)
3. Verify that Functional Tests suite execution time is similar to the one for `2.3-develop` branch

### Contribution checklist
